helpers/test-helper.js

```diff
@@ -1144,8 +1144,6 @@ exports.getWaitForNSubmissionsPromise = async function getWaitForNSubmissionsPro
       if (nSubmissions.toNumber() >= n) {
         event.removeListener();
         resolve();
-      } else {
-        await exports.mineBlock();
```

Comment:
Where did this go?

Reply:
Well, it shouldn't be necessary, and extra (quasi-unexpected) blocks being mined makes it harder to pin down what exactly is going on with these tests. It's possible that removing this will have made other tests spottier, though.